src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceSecurity.cs

```diff
@@ -145,11 +145,6 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
 
         private IntPtr GetMainWindowHandle()
```

Comment:
Since we now always prompt using console we can get rid of this method since it will never be used.

Reply:
PromptForCredential() can be simplified since it now always prompts from console.  Also we can remove the HostUtilities.CredUIPromptForCredential() method since it also will no longer be used.

Reply:
Agree, will remove